storybook/main.cpp

```diff
@@ -8,9 +8,9 @@ int main(int argc, char *argv[])
     QCoreApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #endif
     QGuiApplication app(argc, argv);
-    app.setOrganizationName("Status");
-    app.setOrganizationDomain("status.im");
-    app.setApplicationName("Status Desktop Storybook");
+    QGuiApplication::setOrganizationName("Status");
```

Comment:
Is it better then the previous version? I'm just curios :)

Reply:
This is what clang-tidy suggested to me: https://clang.llvm.org/extra/clang-tidy/checks/readability/static-accessed-through-instance.html